mitodl/micromasters
# Use babel-plugin-istanbul to fix coverage

#### What are the relevant tickets?
Fixes #2638 

#### What's this PR do?
Uses babel-plugin-istanbul to provide coverage for transformed code.

#### How should this be manually tested?
Run `docker-compose run watch npm run coverage` and look at the results in `coverage/`. The covered lines should match up correctly

#### Any background context?
We could use sourcemaps to fix the stack traces when errors occur but this slows the tests down significantly, so I left it out. We still have correct stack traces anyway when running without coverage.
